annotation/annotation-experimental-lint/src/main/java/androidx/annotation/experimental/lint/ExperimentalIssueRegistry.kt

```diff
@@ -23,7 +23,7 @@ import com.android.tools.lint.detector.api.CURRENT_API
 @Suppress("UnstableApiUsage")
 class ExperimentalIssueRegistry : IssueRegistry() {
```

Comment:
If we need them somewhere - yes, sure. But I don't see how it's related to `experimental/lint` file

Reply:
I attached my comment to the random line 🫠. I was talking about

replacing
```
implementation("org.jetbrains.compose.annotation-internal:annotation:1.7.1")
```
by
```
project(":annotation:annotation")
```

In the whole project.

Reply:
I don't think it's necessary without an explicit need for it. If we stop publish `annotation-internal`, it will be a good timing for it. But unpinning is not related to the update version itself.

Reply:
>  an explicit need for it

Okay. As far as I remember, this change adds some needed implementation for ìntegration` branch.

If it is needed for it, we can unpin there.